mongoengine/document.py

```diff
@@ -503,6 +503,10 @@ def switch_db(self, db_alias):
 
         :param str db_alias: The database alias to use for saving the document
 
+        :param bool keep_created: keep self._created value after call `swith_db()` when True,
```

Comment:
From this description I can't understand what the argument does and why it's needed.